app/index.js

```diff
@@ -190,14 +194,35 @@ module.exports = yeoman.Base.extend({
     },
 
     composerJson: function () {
-      var composer = this.fs.readJSON('composer.json');
+      // Import any existing composer.json that was saved above.
+      var composer = {};
+      var composerExists = this.fs.exists('composer.orig');
```

Comment:
Changing the above to load json values into memory instead of using the virtual file should eliminate the need to use fs.exists.